helm-chart/values.yaml

```diff
@@ -28,12 +28,11 @@ tap:
   dryrun: false
   ignoretainted: false
   ingress:
-    certmanager: letsencrypt-prod
-    classname: kubeshark-ingress-class
-    controller: k8s.io/ingress-nginx
     enabled: false
-    host: ks.svc.cluster.local
-    tls: null
+    # ingressClassName: nginx
```

Comment:
```suggestion
    ingressclass: kubeshark-ingress-class
```

Reply:
That ingress-class does not exist anymore. The default value should be empty, so the cluster will use the default ingress class of the cluster.